src/python/tests/test_orthology_benchmark.py

```diff
@@ -301,3 +301,12 @@ def test_prepare_gtf_files(core_names: str, tmp_dir: Path, expectation: ContextM
         exp_out2 = rel_dir / "metazoa" / "gtf" / "anopheles_albimanus" / "Anopheles_albimanus.AalbS2.51.gtf"
         assert file_cmp( tmp_dir / "Juglans_regia.Walnut_2.0.51.gtf", exp_out1)
         assert file_cmp( tmp_dir / "Anopheles_albimanus.AalbS2.51.gtf", exp_out2)
+
+
+def test_extract_orthologs() -> None:
+    """Tests :func:`orthology_benchmark.extract_orthologs()` function.
+    """
+    # pylint: disable-next=no-member
+    test_files_dir = pytest.files_dir / "orth_benchmark"  # type: ignore[attr-defined]
+    assert orthology_benchmark.extract_orthologs(test_files_dir , "gallus_gallus", "homo_sapiens") == \
+           [("ENSGALG00000030005", "ENSG00000147255")]
```

Comment:
I would suggest to use the backtick sparingly for line continuation. You could break the line at the start of the expected output list:
```suggestion
    assert orthology_benchmark.extract_orthologs(test_files_dir, "gallus_gallus", "homo_sapiens") == [
        ("ENSGALG00000030005", "ENSG00000147255")
    ]
```
Alternatively, you could assign the observed and expected output to variables and compare these:
```suggestion
    obs_output = orthology_benchmark.extract_orthologs(test_files_dir, "gallus_gallus", "homo_sapiens")
    exp_output = [("ENSGALG00000030005", "ENSG00000147255")]
    assert obs_output == exp_output
```
In fact, there are multiple ways to write this `assert` statement that would all make `pylint` happy. So in this case, I would ask, which of these alternatives would best communicate what is happening to my human colleagues?